airbyte-integrations/bases/base-normalization/normalization/transform_config/transform.py

```diff
@@ -224,6 +223,17 @@ def transform_snowflake(config: Dict[str, Any]):
             "connect_retries": 3,
             "connect_timeout": 15,
         }
+
+        credentials = config.get("credentials", {})
+        if credentials.get("auth_type") == "OAuth2.0":
+            dbt_config["authenticator"] = "oauth"
+            dbt_config["oauth_client_id"] = credentials["client_id"]
+            dbt_config["oauth_client_secret"] = credentials["client_secret"]
+            dbt_config["token"] = credentials["refresh_token"]
+        elif credentials:
```

Comment:
shouldn't this check for `credentials["password"]` ?

Reply:
I have improve to be more explicit